google-oauth-client/src/main/java/com/google/api/client/auth/oauth/OAuthParameters.java

```diff
@@ -14,16 +14,14 @@
 
 package com.google.api.client.auth.oauth;
 
-import com.google.api.client.http.GenericUrl;
-import com.google.api.client.http.HttpExecuteInterceptor;
-import com.google.api.client.http.HttpRequest;
-import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.*;
```

Comment:
Google doesn't use wildcard imports

Reply:
I have removed the wildcard import in my most recent commit.  Regarding an issue to explain the bug, a pre-existing issue was referenced in the pull request description.

Fixes #1